CONTRIBUTING.md

```diff
@@ -1,15 +1,42 @@
-# Sanoma Learning Design System contributing guide
-
-We would like to welcome all contributors.
-
-These are the basic contributing rules:
-- New branch should be created from the `main` branch
-- Branch naming: `feature/issueNumber-...-...` or `fix/issueNumber-...-...`, eg `feature/21-button-component`
-- Please use English for naming 🙂
-- Every component should have prefix `sl-` eg. `sl-button`
-- Please add unit tests for your changes
-- Commit message with scope (eg. button, checkbox) for instance:
-    `(button): Create button component`
-- Before creating PR – the newest version of `main` branch should be merged into feature branch and all conflicts should be resolved
-- Squash before merge (GitHub  will do squash for you)
-- Not merging PR without code review
\ No newline at end of file
+# Contributing
+
+Thanks for wanting to contribute to the project!
+
+## Getting Started
+
+- Ensure you have the LTS version of Node.js installed (see https://nodejs.org/en/)
+- Ensure if you are using Windows to use the Windows Subsystem for Linux (WSL)
+- Please be respectful when communicating within the project
+
+## Naming
+
+This is an international project and as such everything uses the English language.
+
+When writing code in this project, please follow the following naming conventions:
+- For NPM packages, we use the `sanomalearning` organization with the `slds-` prefix for the actual packages; for example: `@sanomalearning/slds-core`
+- For web components, we use the `sl-` prefix; for example: `<sl-button>`
+- All web component events use the `sl-` prefix and "dasherize" convention; for example: `sl-active-item-change`
+
+## Branching
+
```

Comment:
Will we add some information about branch naming?

Reply:
Good idea.